script/deploy_snapshot.sh

```diff
@@ -18,6 +18,6 @@ elif [[ $(./gradlew properties | grep version) != *-SNAPSHOT ]]; then
   echo "Skipping snapshot deployment: not a snapshot version."
 else
   echo "Deploying snapshot..."
-  ./gradlew :GooglePayment:uploadArchives
+  ./gradlew :GooglePayment:publishToSonatype
```

Comment:
Snapshot build scripts will be purged at a later date across all repos. Making the change here for due diligence.